.github/workflows/release.yml

```diff
@@ -9,6 +9,39 @@ permissions:
   contents: read
 
 jobs:
+
+  publish-docker-images:
+    strategy:
+      matrix:
+        include:
+          - service: coupon
+          - service: frontend
+          - service: inventory
+          - service: membership
+          - service: pricing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Store new version in env
+        run: echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
Tagging with `${GITHUB_REF#refs/*/}` fails if branch name contains a `/` - I prefix my branches with `feat/` or `fix/`. Use `${GITHUB_REF##refs/*/}` to strip that as well.

Reply:
The workflow should run for version tags only: 

```
on:
  push:
    tags:
      - "v*"
```

So I think it should be ok, and this is also how it's used in all the other workflow jobs.

But good to know :)